.github/workflows/bokeh-ci.yml

```diff
@@ -183,9 +183,9 @@ jobs:
       - name: Prepare Environment
         uses: ./.github/workflows/composite/test-setup
         with:
-          test-env: '3.11'
-          source-tree: 'delete'
-          sampledata: 'cache'
+          test-env: ["3.11", "3.12-dev"]
```

Comment:
This isn't valid, it is expecting a single value here not a sequence; see https://github.com/bokeh/bokeh/actions/runs/5032752743. These `integration-tests` are currently disabled anyway (see line 176 above) so changes in this section won't have any effect.

Actually, please can you revert all changes to this file. We don't want to attempt testing on python `3.12-dev` in CI as most of our dependencies do not exist on Python 3.12 yet. Plus your code editor (or similar) has changed the formatting in this file, which aren't changes we want to see in an unrelated PR.

Reply:
For sure! I had noticed this when I was trying to setup my environment that those variable values had changed. Reverting this back to the previous version.

Regarding the formatting, I'm using vscode on mac. Do you suggest any settings I should change?

Reply:
I suggest you turn off any automatic formatting that you have enabled when you are making code changes. They are fine to have enabled for your own work, but when submitting changes to Open Source projects you don't want to change any other lines unnecessarily.